lib/iris/aux_factory.py

```diff
@@ -566,9 +449,7 @@ def calc_bounds():
                     orography = orography_pts.reshape(
                         orography_pts_shape.append(1))
                 return self._derive(delta, sigma, orography)
-            b_shape = self._shape(nd_values_by_key)
-            b_dtype = self._dtype(nd_values_by_key)
-            bounds = _LazyArray(b_shape, calc_bounds, b_dtype)
+            bounds = calc_bounds()
```

Comment:
OK, maybe it isn't so bad pulling the code inline, but it still makes for a very long `make_coord` method. I wonder whether it would be better to pull the bounds calculation code up into a new method in each case, or whether to just leave that problem for the future. @pp-mo @lbdreyer thoughts?